Document new fine grained controll over distributed tracing (#7757)
getsentry/sentry-docs: src/platforms/python/guides/celery/index.mdx

  See <PlatformLink to="/crons/#celery-beat-auto-discovery">Celery Beat Auto Discovery</PlatformLink> to learn more.

  The default is `None`.

## Distributed Traces

> Distributes tracing means, that the code running in your Celery tasks will be part of the trace that was started in the code that started your Celery task.

Reviewer: ```suggestion
Distributed tracing extends the trace from the code that's running your Celery task so that it includes the code that initiated the task.
```

Hopefully I have the mental model right here :)

> You can disable distributing the tracing information into your Celery tasks globally with the `propagate_traces` parameter documented above. If you set `propagate_traces` to `False` all Celery tasks will start their own new trace.

Reviewer: ```suggestion
You can disable this globally with the `propagate_traces` parameter, documented above. If you set `propagate_traces` to `False`, all Celery tasks will start their own trace.
```

> If you want to have more fine grained control over trace distribution, you can overide the `propagate_traces` option by passing the `sentry-propagate-traces` header when starting the Celery task:

Reviewer: ```suggestion
If you want to have more fine-grained control over trace distribution, you can override the `propagate_traces` option by passing the `sentry-propagate-traces` header when starting the Celery task:
```
